response_operations_ui/controllers/survey_controllers.py

```diff
@@ -30,3 +30,15 @@ def get_survey(short_name):
 
     logger.debug('Successfully retrieved survey', short_name=short_name)
     return response.json()
+
+
+def get_survey_short_name_by_id(survey_id):
+    # TODO cache the surveys dictionary at app start up
+    try:
+        surveys_dict = {survey['id']: survey for survey in get_surveys_list()}
+        return surveys_dict[survey_id]['shortName']
+    except ApiError:
+        return 'Unavailable'
+    except KeyError:
+        logger.exception("failed to resolve survey short name", survey_id=survey_id)
+        return 'Unavailable'
```

Comment:
I think this dismissed my last statement - is this correct in the UI when returning unavailable?